content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -5,49 +5,57 @@ weight: 12
 keywords: [kubernetes,alicloud,aliyun]
 ---
 
-Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+Follow these instructions to install and run Istio in the 
+[AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) 
+using the `Application Catalog`.
 
-This Quick Start installs the current release version of Istio and then deploys the [Bookinfo](/docs/examples/bookinfo/) sample
-application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+This guide installs the current release version of Istio and deploys the 
+[Bookinfo](/docs/examples/bookinfo/) sample application.  
+To automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/)
```

Comment:
This sentence is incomplete. There is a missing action. What does the reader need to do to automate ...?

Reply:
agree, remove this wordy sentence

Reply:
see the new commit 4ab4781